pro/src/pages/Offers/Offers/OfferItem/Cells/CollectiveActionsCells.tsx

```diff
@@ -75,6 +80,22 @@ export const CollectiveActionsCells = ({
 
   const isMarseilleActive = useActiveFeature('WIP_ENABLE_MARSEILLE')
 
+  const startDatetime = offer.stocks[0].beginningDatetime
```

Comment:
Ce `beginningDatetime` me perturbe, c'est lié à ce qu'on a noté hier en atelier tech dans [ce ticket ](https://passculture.atlassian.net/browse/PC-30571) ?